pkg/curatedpackages/kubeclient.go

```diff
@@ -0,0 +1,12 @@
+package curatedpackages
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/executables"
+)
+
+type KubeClient interface {
+	ApplyResources(ctx context.Context, opts ...executables.KubectlOpt) (bytes.Buffer, error)
```

Comment:
This interface is a bit problematic because it receives a concrete type from the `executables` package. That creates a dependency on `executables` and break the abstraction.

I'm ok addressing this in separate PR if we come up with a solution and create a ticket :)